chainstate/src/detail/median_time.rs

```diff
@@ -185,11 +186,11 @@ mod test {
             .unwrap();
 
             // we use unordered block times, and ensure that the median will be in the right spot
-            let block1_time = current_time.load(Ordering::SeqCst) + 1;
-            let block2_time = current_time.load(Ordering::SeqCst) + 10;
-            let block3_time = current_time.load(Ordering::SeqCst) + 17;
-            let block4_time = current_time.load(Ordering::SeqCst) + 18;
-            let block5_time = current_time.load(Ordering::SeqCst) + 20;
+            let block1_time = current_time.load() + 1;
+            let block2_time = current_time.load() + 20;
```

Comment:
Sorry if this was answered already, I didn't keep up with all the PRs. Why did the tests need changes here?

Reply:
Sam asked me to do this: https://github.com/mintlayer/mintlayer-core/pull/984#discussion_r1237864249

Reply:
Perfect, thanks